CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-06-09
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as v20240609. 
+
+### Announcements
+* Starting 1.30 Kubernetes version and 1.27 LTS versions, beta apis will be disabled by default, when you upgrade to [them](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#before-you-begin). There will be an option provided to explicitly enable beta apis closer to the 1.30 release.
+
+
+### Release Notes
+
+* Features:
+  * Generally Available - [Security Patch channel - Live patching mechanism](https://learn.microsoft.com/azure/aks/auto-upgrade-node-os-image?tabs=azure-cli) for VHD updates.
+  * AKS Patch version [1.27.13](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md#v12713) is now available. 
+
+* Preview Features:
+  * AKS version [1.30](https://kubernetes.io/blog/2024/04/17/kubernetes-v1-30-release/) is available in preview. 
+
+* Behavioral Changes:
+  * asm-1-19 is no longer supported. If this revision is in use, please upgrade for continued support. More information about mesh upgrades and version support can be found [here](https://aka.ms/asm-aks-upgrade-docs).
+
+* Bug Fixes:
+  * Updating coredns to use image [v1.9.4-hotfix.20240520](https://github.com/aks-lts/coredns/tree/release-1.9) on all AKS clusters with version 1.24+. We still use coredns build version v1.9.4, we have only fixed the [CVE vulnerabilities](https://github.com/aks-lts/coredns/commit/ba698d28c2ab8d9db0951592be631885e4134e5) on this.
+  * Replacing cilium 1.14.4 with new patch 1.14.10 for k8s 1.29+ fixes the [issue](https://github.com/cilium/cilium/issues/18706).
+  * Removes the post-upgrade annotation on [hubble-generate-cert](https://github.com/cilium/cilium/blob/aa10df3a4c6a9e7bd947a4a32613cedf22b3731d/Documentation/gettingstarted/hubble-configuration.rst#L81) Job. On each aks cluster reconcile, the helm chart revision is incremented which counts as an upgrade. Each time the   helm chart is upgraded or installed this job will restart. This change fixes that to not restart on helm chart upgrades and successfully clean up.
+   * Bumped Windows containerd v1.7.14 to v1.7.17 in k8s v1.28+. Fixes issues [4196](https://github.com/Azure/AKS/issues/4196) and [72](https://github.com/containerd/ttrpc/issues/72#issuecomment-2105545516).
```

Comment:
Bumped Windows containerd v1.7.14 to v1.7.17 in k8s v1.28+. Fixes issues [4196](https://github.com/Azure/AKS/issues/4196) and [72](https://github.com/containerd/ttrpc/issues/72#issuecomment-2105545516).

to

Windows containerD has been upgraded from v1.7.14 to v1.7.17 in K8s v1.28+. This upgrade fixes two bugs resulting in a [wrong default path]((https://github.com/Azure/AKS/issues/4196) and a [deadlock issue](https://github.com/containerd/ttrpc/issues/72).